tests/cases/fourslash/completionsWithOptionalProperties.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="fourslash.ts" />
+// @strict: true
```

Comment:
Wait, is this how options flags are specified in fourslash @sheetalkamat @RyanCavanaugh @Kingwl?

Reply:
Yes